neo/SmartContract/Native/NativeContractBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Neo.VM;
+
+namespace Neo.SmartContract.Native
+{
+    public abstract class NativeContractBase
+    {
+        protected static byte[] CreateNativeScript(string serviceName)
```

Comment:
Perhaps the `ScriptHash => Hash of name` part should be here as well.

Reply:
`ScriptHash` is the hash of the script, not the name.

Reply:
I wrote this in a rush... I mean, the hash of the script that represents the name...